### DISCUSSION TITLE
关于杜比大喇叭的问题

### DISCUSSION THREAD

**Discussion by Extreme-Icer** [1715173329/UnblockNeteaseMusic#1079] [Q&A] [answered]
杜比大喇叭众所周知基本停更了，最近倒是有一次更新但是没什么用，于是我想了一下将里面的unblock脚本同步到本仓库最新的版本，这样一来匹配就不会报错了。但是现在的问题是，网易云一旦APP版本高于8.4.55就会提示找不到Cookie类并闪退，这要如何解决呢？
替换脚本zip不难，主要是要fork一份到自己仓库，上传新处理过的zip然后重写action脚本，手动发起构建。

**Comment by 1715173329** [comment]
你应该去问上游

**Reply by Extreme-Icer** [reply]
> 你应该去问上游

那就是说明这个是模块问题，这个脚本不会导致cookie类找不到，需要修改模块相关的代码吧。

**Reply by 1715173329** [reply, marked as answer]
对
